modules/ossm-federation-config-export.adoc

```diff
@@ -0,0 +1,152 @@
+* You can use wildcards to select services, for example,  to export all services in a namespace.
+* You can select services for export using a label selector, which may  be global to the mesh, or scoped to a specific member namespace.
+* You can export services using an alias, for example, you can export the `foo/bar` service as `custom-ns/bar`.
+// Need non foo/bar example above
+* You can only export services that are visible to the mesh’s system namespace.  For example, a service in another namespace with an `networking.istio.io/exportTo` label set to ‘.’ would not be a candidate for export.
+* For exported services, their target services will only see traffic from the ingress gateway, not the original requestor (that is, they won’t see the client ID of either the other mesh’s egress gateway or the workload originating the request)
+
+The following example is for services that `red-mesh` is exporting to `green-mesh`.
+
+.Example ExportServiceSet resource
+[source,yaml]
+----
+kind: ExportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: green-mesh
+  namespace: red-mesh-system
+spec:
+  exportRules:
+  # export ratings.mesh-x-bookinfo as ratings.bookinfo
+  - type: NameSelector
+    nameSelector:
+      namespace: red-mesh-bookinfo
+      name: red-ratings
+      alias:
+        namespace: bookinfo
+        name: ratings
+# export any service in red-mesh-bookinfo namespace with label export-service=true
+  - type: LabelSelector
+    labelSelector:
+      namespace: red-mesh-bookinfo
+      Selector:
+        matchLabels:
+          export-service: “true”
+      alias: # exported as if they were in the bookinfo namespace
+        namespace: bookinfo
+----
+
+.ExportServiceSet parameters
+[options="header"]
+[cols="l, a, a"]
+|===
+|Parameter |Description |Values
+|metadata:
+  name:
+|Name of the ServiceMeshPeer you are exposing this service to.
+|Must match the `name` value for the mesh in the `ServiceMeshPeer` resource.
+
+|metadata:
+  namespace:
+|Name of the project/namespace you are exporting this service from, that is, a namespace that is listed on this service mesh's member roll (SMMR) .
+|
+
+|spec:
+  exportRules:
+  -type:
+|Type of rule that will govern the export for this service.  The first matching rule found for the service will be used for the export.
+|`NameSelector`, `LabelSelector`
+
+|spec:
+  exportRules:
+  -type: nameSelector
+    NameSelector:
+      namespace:
+      name:
+|To create a `NameSelector` rule, specify the `namespace` of the service and the `name` of the service as defined in the `Deployment` resource.
+|
+
+|spec:
+  exportRules:
+  -type: NameSelector
+    NameSelector:
+      alias:
+        namespace:
+        name:
+|To create a `NameSelector` rule that uses an alias for the service, after specifying the `namespace` and `name` for the service, then specify the alias for the `namespace` and the alias to be used for `name` of the service.
```

Comment:
```suggestion
|To create a `NameSelector` rule that uses an alias for the service, specify the `namespace` and `name` for the service. Then specify the alias for the `namespace` and the alias to be used for `name` of the service.
```

Reply:
Is there some way to make this phrase simpler and clearer? While reading it, my attention snags on "after." It wonders, does after signal the position of what I'm reading relative to the first part (to create) or the last part (then specify)? For example, am I reading something like, "To change your tire, after hitting a sharp object, do such and such,"  OR "To change your timer, do a, then do b."